Update/omot_010_2.txt

```diff
@@ -53,7 +53,7 @@ void main()
 //r…おかしい。確かに聞かされて楽しい内容ではないが、…それにしてもレナの反応は、妙に排他的だ。だから私は思わず、余計かもと思ったがつい、口を挟んでしまった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "…おかしい。確かに聞かされて楽しい内容ではないが、…それにしてもレナの反応は、妙に排他的だ。だから私は思わず、余計かもと思ったがつい、口を挟んでしまった。",
-			NULL, "...That's strange. ...It's certainly not a fun thing to hear, but... Rena's reaction is oddly combative. So, I chimed in without thinking.", Line_Normal);
+			NULL, "...That's strange. ...It's certainly not a fun thing to hear, but... Rena's reaction is oddly combative. So, I reflexively chimed in even though I thought it might be uncalled-for.", Line_Normal);
```

Comment:
```suggestion
			NULL, "...That's strange. ...It's certainly not a fun thing to hear, but... Rena's reaction is oddly combative. So I reflexively chimed in, even though I thought it might be uncalled-for.", Line_Normal);
```